SU2_CFD/include/solvers/CFEASolver.hpp

```diff
@@ -28,6 +28,7 @@
 #pragma once
 
 #include "CSolver.hpp"
+#include "../../../Common/include/geometry/elements/CElement.hpp"
 #include "../../../Common/include/omp_structure.hpp"
```

Comment:
What happened here? In any case, always try to include only in cpp's, if that fails because the interface of CFEASolver mentions pointer or references to CElement, forward declare in the hpp.

Reply:
A couple of changes led to me adding the CElement include here. First,  `CFEASolver` defines 
```
CElement*** element_container = nullptr;  
CProperty** element_properties = nullptr;
```
in its header file. So it needs `CElement` included in the header file. Prior to this, `CElement` was included in `grid_movement_structure.hpp` which was included in `CFEASolver` through the `CSolver.hpp` include. But `CElement` isn't needed for any of the grid movement classes and, out of the solvers, it is only needed for the `CFEASolver` and `CMeshSolver`. Hence the new include here.

Reply:
Ah I see, I forgot to delete that include from the grid_movement when I removed some of the "intermediate legacy" mesh deformation.